### PR TITLE
PBM-646: Stop the balancer during backup

### PR DESCRIPTION
**Pull request by dAdAbird**
https://jira.percona.com/browse/PBM-646

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| Restore on new cluster mongodb 3.6 | passed |
| Restore on new cluster mongodb 4.0 | passed |
| Restore on new cluster mongodb 4.2 | passed |
| Sharded cluster mongodb 4.0 | failed |
| Sharded cluster mongodb 3.6 | failed |
| Non-sharded replicaset mongodb 4.0 | passed |
| Sharded cluster mongodb 4.2 | failed |
| Non-sharded replicaset mongodb 3.6 | passed |
| Non-sharded replicaset mongodb 4.2 | passed |

File: pbm/backup/backup.go [hunk 1]
```diff
@@ -128,6 +135,23 @@ func (b *Backup) run(ctx context.Context, bcp pbm.BackupCmd, opid pbm.OPID, l *p
 				l.Info("mark backup as %s `%v`: %v", status, err, ferr)
 			}
 		}
+
+		// Eurn on the balancer back if needed
+		//
+		// Every agent will check if the balancer was on before the backup started.
+		// And will try to turn in on if so. So if the leader node went donw after turning off
+		// the ballancer some other node will bring it back.
```
Comment (akira-kurogane) [inline pbm/backup/backup.go:143]:
```suggestion
		// Turn the balancer back on if needed
		//
		// Every agent will check if the balancer was on before the backup started.
		// And will try to turn it on again if so. So if the leader node went down after turning off
		// the balancer some other node will bring it back.
```

**Review by akira-kurogane — APPROVED** [review]
:+1:

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

**Pull request merged**